doc/source/tutorial/csgraph.rst

```diff
@@ -47,9 +47,9 @@ later::
     >>> word_list = [word for word in word_list if len(word) == 3]
     >>> word_list = [word for word in word_list if word[0].islower()]
     >>> word_list = [word for word in word_list if word.isalpha()]
-    >>> word_list = map(str.lower, word_list)
+    >>> word_list = list(map(str.lower, word_list))
     >>> len(word_list)
-    586
+    586    # may vary
```

Comment:
It depends on how long the word list on any given computer. I have 1135 three letter words on my Mac.